### PR TITLE
I18n #1

### PR DESCRIPTION
**Pull request by yanarchy**
Part of #1 

Added:

* LocaleSwitcher to go back and forth
* A configs file has the two languages
* Connected to Toolbar---clicking changes languages

### CODE REVIEW

File: web/src/components/HelloWorld.vue [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<template>
```
Comment (ann-kilzer) [inline web/src/components/HelloWorld.vue:1]: We'd deleted this file awhile ago. Let's remove it.

File: web/src/components/LocaleSwitcher.vue [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<template>
+  <v-select
+    v-model='$i18n.locale'
+    prepend-icon='fa-globe'
```
Comment (ann-kilzer) [inline web/src/components/LocaleSwitcher.vue:4]: Let's use material icons for consistency. I recommend the ['language' icon](https://material.io/resources/icons/?icon=language&style=baseline).

File: web/yarn.lock [hunk 3]
```diff
@@ -804,6 +804,11 @@
     lodash "^4.17.13"
     to-fast-properties "^2.0.0"
 
+"@fortawesome/fontawesome-free@^5.13.0":
```
Comment (ann-kilzer) [inline web/yarn.lock:807]: (As mentioned previously) Let's stick with material icons for now.

File: web/src/views/Nav.vue [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<template>
```
Comment (ann-kilzer) [inline web/src/views/Nav.vue:1]: This looks like a duplicate of Toolbar.vue. Are you planning on merging them together?

File: yarn.lock [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (ann-kilzer) [inline yarn.lock:1]: Accidental checkin?

File: web/.env [hunk 6]
```diff
@@ -0,0 +1,2 @@
+VUE_APP_I18N_LOCALE=en
+VUE_APP_I18N_FALLBACK_LOCALE=en
```
Comment (tuttiq) [inline web/.env:2]: `.env` was added to `.gitignore` a while ago, should this be in the sample file instead (`.env.sample`)?

File: web/.gitignore [hunk 7]
```diff
@@ -5,6 +5,7 @@ node_modules
 # local env files
 .env.local
 .env.*.local
+.env
```
Comment (tuttiq) [inline web/.gitignore:8]: I thought this was already here 🤔  but if not, it should be! lol and `.env` above should not be committed

File: web/src/components/HelloI18n.vue [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<template>
```
Comment (tuttiq) [inline web/src/components/HelloI18n.vue:1]: Is this a test file or being used?

File: web/src/components/LocaleSwitcher.vue [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+import supportedLocales from '@/config/supported-locales';
+
+const { en, ja } = supportedLocales;
+
+export default {
+  data: () => ({ locale: en }),
+  methods: {
+    changeLocale() {
+      this.locale = (this.locale === en) ? ja : en;
+      // this.$router.push(`/${this.locale}`);
```
Comment (ann-kilzer) [inline web/src/components/LocaleSwitcher.vue:22]: Let's avoid committing commented-out code into master.

File: web/src/components/LocaleSwitcher.vue [hunk 10]
```diff
@@ -0,0 +1,27 @@
+<template>
+  <v-btn
+    v-model="$i18n.locale"
```
Comment (ann-kilzer) [inline web/src/components/LocaleSwitcher.vue:3]: Thanks for clarifying the language => locale 😄

File: web/src/config/supported-locales.js [hunk 11]
```diff
@@ -0,0 +1,4 @@
+export default {
```
Comment (ann-kilzer) [inline web/src/config/supported-locales.js:1]: Should we use a camelCase name on this file to match the Vue file conventions?

File: web/src/main.js [hunk 12]
```diff
@@ -5,6 +5,7 @@ import vuetify from './plugins/vuetify';
 import Airtable from './plugins/airtable';
 import 'roboto-fontface/css/roboto/roboto-fontface.css';
 import '@mdi/font/css/materialdesignicons.css';
+import i18n from './util/i18n';
```
Comment (ann-kilzer) [inline web/src/main.js:8]: I think this directory didn't get committed in. Can you please add it?

File: web/src/config/supported-locales.js (hunk 11 above)
Comment (ann-kilzer) [reply web/src/config/supported-locales.js:1]: nm, I see this is all generated by vue-cli

**Pull request merged**